sections/04_study.md

```diff
@@ -107,23 +108,23 @@ immune diseases, implicating mis-splicing as a much more widespread feature of
 human pathologies than previously thought [@tag:Li2016_variation].
 
 Sequencing studies routinely return thousands of unannotated variants, but which
-cause functional changes in splicing, and if so, how? Prediction of a “splicing
-code” has been a holy grail over the past decade. Initial machine learning
+cause functional changes in splicing and how are those changes manifested? Prediction of a "splicing
+code" has been a holy grail over the past decade. Initial machine learning
```

Comment:
holy grail is loaded - do we want to revise to something a bit less provocative here?